app/helpers/date_helper.rb

```diff
@@ -1,4 +1,8 @@
 module DateHelper
+  def format_date(date)
```

Comment:
The argument could probably be `datetime`

Reply:
Have changed to datetime for this and other methods for clarity